package.json

```diff
@@ -12,7 +12,7 @@
     "prop-types": "15.7.2",
     "react": "17.0.2",
     "react-dom": "17.0.2",
-    "react-leaflet": "^2.7.0",
+    "react-leaflet": "^2.8.0",
```

Comment:
Seems to work ok from my test, im still not quite sure why this bump was helpful for you.

Reply:
ah i see now, it allows you to avoid the peer dependencies problem